pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.builds

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Importing configurations.props will get us the RuntimeOS, but will default our ConfigurationGroup to Debug,
+         so we save the configurationgroup if one was specified. -->
+    <Input_ConfigurationGroup>$(ConfigurationGroup)</Input_ConfigurationGroup>
```

Comment:
I was thinking about it this morning.  I think if we just pass in BuildConfiguration it should work.  That ought to be the behavior for any project that doesn't opt into listing BuildConfigurations anyway.  Perhaps its just that the CI/scripts for building packages forgot to pass this in?